inno_setup.iss

```diff
@@ -202,7 +202,7 @@ begin
   begin
     // Is the installed version at least the packaged one ?
     Log('VC Redist x64 Version : found ' + strVersion);
-    Result := (CompareStr(strVersion, 'v14.38.33130') < 0);
+    Result := (CompareStr(strVersion, 'v14.32.31332') < 0);
```

Comment:
This is also suspicious. Should probably not go back in versions

Reply:
Oh, I didn't know that one was a thing.